src/components/MdComponents.tsx

```diff
@@ -32,44 +34,73 @@ import InfoBanner from "./InfoBanner"
 import Card from "./Card"
 import QuizWidget from "./Quiz/QuizWidget"
 import DocLink from "./DocLink"
+import Link from "@/components/Link"
+import { motion } from "framer-motion"
 // import Contributors from "@/components/Contributors"
 // import Logo from "@/components/Logo"
 // import MeetupList from "@/components/MeetupList"
 
 /**
  * Base HTML elements
  */
-export const commonHeadingProps: HeadingProps = {
+const headingPropsForAnchor = (id?: string): HeadingProps =>
+  id
+    ? ({
+        scrollMarginTop: 28,
+        id,
+        "data-group": true,
+        position: "relative",
+      } as HeadingProps)
+    : {}
+
+export const commonHeadingProps = (id?: string): HeadingProps => ({
   fontWeight: 700,
   lineHeight: 1.4,
-}
+  ...headingPropsForAnchor(id),
+})
 
-export const Heading1 = (props: HeadingProps) => (
-  <OldHeading as="h1" {...commonHeadingProps} fontSize="2.5rem" {...props} />
+const IdAnchor: React.FC<{ id?: string }> = ({ id }) =>
+  id ? (
+    <Link href={`#${id}`}
+      position="absolute"
+      insetInlineEnd="100%"
+    >
+      <Icon
+        as={CiLink}
+        opacity={0}
+        _groupHover={{ opacity: 1 }}
+        transition="opacity 0.1s ease-in-out"
+        fontSize="xl"
+        me={1}
+      />
+    </Link>
+  ) : null
+
+export const Heading1 = ({ children, ...rest }: HeadingProps) => (
+  <OldHeading as="h1" {...commonHeadingProps()} fontSize="2.5rem" {...rest}>
+    {children}
+  </OldHeading>
 )
```

Comment:
Left autolinking off for H1s as these are the top of the page (parity with Gatsby site)